make lint dependencies missing in CLI dev Docker containers

Inside `skycoin/skycoindev-cli` image .

```sh
$ make lint
vendorcheck ./...
golangci-lint run --no-config --deadline=3m --disable-all --tests --skip-dirs=lib/cgo \
	-E golint \
	-E goimports \
	-E varcheck \
	-E unparam \
	-E deadcode \
	-E structcheck \
	-E errcheck \
	-E gosimple \
	-E staticcheck \
	-E unused \
	-E ineffassign \
	-E typecheck \
	-E gas \
	-E megacheck \
	-E misspell \
	-E nakedret \
	./...
make: golangci-lint: Command not found
Makefile:124: recipe for target 'lint' failed
make: *** [lint] Error 127
```

--- a/docker/images/dev-cli/Dockerfile
+++ b/docker/images/dev-cli/Dockerfile
@@ -196,6 +196,10 @@ RUN cd /tmp/; \
     ./configure --prefix=/usr && make && make install && \
     rm -rf /tmp/swig-*
 
+# Install golangci-lint
+ENV GOLANGCI_LINT 1.10.2
+RUN curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v$GOLANGCI_LINT
+
 WORKDIR $GOPATH/src/github.com/skycoin
 VOLUME $GOPATH/src/
 

